dev-util/flatpak-builder/flatpak-builder-1.4.3.ebuild

```diff
@@ -20,7 +22,9 @@ RDEPEND="
 	>=dev-libs/libxml2-2.4:=
 	dev-libs/json-glib:=
 	net-misc/curl:=
+	dev-util/debugedit:=
```

Comment:
The `:=` slot operator is generally reserved for soname dependencies, so we can omit it here.